### ISSUE TITLE
Struggling to get a Clover USB to boot fully.

### ISSUE THREAD

**Issue by MarkMorrow** [shmilee/T450-Hackintosh#17]
My T450 appears to be a near match to what was used here, so I expected to follow the steps and share success, but so far that is not the case.  A clover v3625 USB gets several pages into its verbose boot of OSX 10.11 before stops responding, right after ethernet initiates.  I often get a spinning color wheel that can be moved with the trackpad, but nothing more.  If I disconnect the ethernet cable I see brief read/write activity on the USB so I suspect it is waiting on something.  Perhaps the setup screen wants input, but is just not showing me the screen.  Suggestions?
## Additional details, and thoughts ...

One difference is my BIOS which is v1.25 instead of v1.18.  Unfortunately, Lenovo says it can't be downgraded.  Has anyone else been able to boot Clover from a T450 from this version?

Based on comments on tonymacx86.com, I chose Clover v3625.  I also tried v3333 and v3751, but got no better results from them.  Is there a preferred versions?  I'd prefer El Capitan, but have tried the latest of both 10.10 and 10.11; downloaded fresh from the App Store, but neither can get past the USB boot.

Win10 always reports, "Dedicated Video Memory: 128MB", regardless of how I set the BIOS's "total graphics memory" setting.  I've focused my tests with the BIOS set to 512mb, per jcsnider's X1 Carbon guide, and have tried the other choice (256mb) without a difference in results.  I've also tried connecting it to external monitors, through both VGA and DP without any difference.  I'm running out of things to try ... and hope.

My Specs:
[http://psref.lenovo.com/Detail/ThinkPad_T450?M=20BV0005US](http://psref.lenovo.com/Detail/ThinkPad_T450?M=20BV0005US)
I've swapped in a 256GB Samsung 850 Pro SSD, and an ADATA 128GB M.2 SSD, with the intent of putting Win10 on the larger drive, and OSX 10.11 on the other.  Win10 went on without an issue, now if only I could coax OSX onto it.

**Comment by tluck** [comment]
when clover comes up, you could try injecting a fake graphics id like 12345678 to avoid trying to load the Intel drivers. it will use the generic VGA to get it booted. as for clover, I am on 3758 (latest is greatest usually).

**Comment by MarkMorrow** [comment]
Thanks for your suggestions and info on clover, **tluck**.  I struggled all weekend with this and just had some success, so I'm very happy now.  I used [**Rehabman**'s instructions](http://www.tonymacx86.com/threads/guide-booting-the-os-x-installer-on-laptops-with-clover.148093/) and his config_HD5300_5500_6000.plist file instead, and was able to get to the graphical "install OS X" screen after only two attempts.  On that second attempt I used fake graphics ID "16160004" (so that helped), but I had been doing that before also when it did not work.  The BIOS version & settings did not change, nor did the Clover version or USB driver used, so at least that answers my questions in that regard.  Now, I only hope that I can get all running stable.  I'm sure **shmilee**'s notes and insight will be of benefit when I start the process of tweaking the setup for that.
